Gemfile.lock

```diff
@@ -74,7 +74,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
I think these lines were unintentionally removed.

Reply:
actually, i think it's because Narin doesn't have the keys setup for sidekiq pro... 

@narin can you do a `git checkout master -- Gemfile.lock` and `git checkout master -- Gemfile` in your branch, then commit those changes? it's annoying, but please make sure not to commit changes to those two files when you commit other changes.

If you add the Sidekiq License to your env, it won't be a problem anymore.

Reply:
Reverted this change